scripts/travis-emscripten/build_emscripten.sh

```diff
@@ -96,6 +96,14 @@ cmake \
 make -j 4
 
 cd ..
+
+md5sum build/libsolc/soljson.js
+
+# Remove debug message "NO_DYNAMIC_EXECUTION was set, using slow cwrap implementation"
+sed -i "s/Runtime\.warnOnce('NO_DYNAMIC_EXECUTION was set,'/if (false) Runtime\.warnOnce('NO_DYNAMIC_EXECUTION was set'/" build/libsolc/soljson.js
```

Comment:
I think this `sed` may not be working. It did on mac, but Linux's sed is different ;(

Reply:
Oh wow! We are patching the binary?!?

Please add a check that the replacement is actually performed. If something in the string changes, we should notice.

Reply:
Ah, that's what the md5 sum is for, ok:)

Reply:
> Oh wow! We are patching the binary?!?

Perhaps its not needed anymore after the emscripten update, checking :)